rero/rero-ils
# test: add cypress tests for document creation

* Add tests to check document creation field by field.
* Moves api requests commands in corresponding resource file.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

To implement new tests on document editor.

## Dependencies

My PR depends on the following PR(s):

* Dev of https://github.com/rero/rero-ils-ui
* https://github.com/rero/ng-core/pull/348

## How to test?

See the documentation to run the tests: https://github.com/rero/developer-resources/blob/master/tests/cypress.md

## Code review check list
